modules/Bio/EnsEMBL/Variation/DBSQL/VariationFeatureAdaptor.pm

```diff
@@ -2464,7 +2470,7 @@ sub fetch_by_spdi_notation{
   # Variation is an indel  
   elsif($check_deleted_seq_letters && $check_inserted_seq_letters){
```

Comment:
Can we support input where the deleted sequence is a length here? Example: NC_000011.9:66321302:2:CA